front-end/src/routes/AppRouter.jsx

```diff
@@ -149,6 +150,18 @@ const RouterWrapper = () => {
             </PrivateRoute>
           }
         />
+        <Route
+          path='/app/subscriptions'
+          element={
+            currentUser?.role === 'student' ? (
+              <PrivateRoute>
+                <StudentSubscriptions />
+              </PrivateRoute>
+            ) : (
+              <div>Teacher Subscriptions</div>
```

Comment:
We already have teachers component. Why you use div?

Reply:
I thought here should be Teacher Subscriptions page